### PR TITLE
javascript-data-model - Jacob Husband

### PR DESCRIPTION
**Pull request by jacobhusband**

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      },
+      {
+        product: "The Art of Sql",
+        item: 2,
+        owner: "Faroult, Stephane",
+        initial_price: 33.99,
+        return_window_date: 20200804,
+      }
+    ]
+    total_price: 138.93,
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:65]: Syntax error here

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
In addition to my other comment, please stick to `camelCase` rather than `snake_case`, it is generally preferred in JavaScript.

**Review by codingcodymiller — APPROVED** [review]
Great work!

**Comment by jacobhusband** [comment]
Thank you!

**Pull request merged**